syntax = "proto3";

package leveldb_cs;


service LevelDBCS {
    rpc Put (PutRequest) returns (PutReply) {
    }
    rpc Get (GetRequest) returns (GetReply) {
    }

}

message PutRequest {
    bytes key = 1;
    bytes value = 2;
}
message PutReply {
    bytes status = 1;
}

message GetRequest {
    bytes key = 1;
}

message GetReply {
    bytes value = 1;
    bytes status = 2;
}